src/dmd/link.d

```diff
@@ -562,59 +562,148 @@ public int runLINK()
             argv.push("-Xlinker");
             argv.push("--gc-sections");
         }
+
+        /// Dynamic array of strings to temporarily hold accumulated,
+        /// reordered linker args
+        Strings linkerArgs;
```

Comment:
This is not used until line 645. Move it to where it is used.

Reply:
The reason for the DDoc comment is it allows users with modern IDE to browse the code and read the comments simply by hovering one's pointer over any usage.

![image](https://user-images.githubusercontent.com/6023641/63630980-e783aa00-c65b-11e9-945a-0da9e3b947f9.png)

But I removed it in https://github.com/dlang/dmd/pull/10341 to avoid any controversy.

Reply:
Addressed in https://github.com/dlang/dmd/pull/10341